python/cugraph/layout/force_atlas2.py

```diff
@@ -109,13 +108,14 @@ def on_train_end(self, positions):
     """
 
     if pos_list is not None:
-        null_check(pos_list["vertex"])
-        null_check(pos_list["x"])
-        null_check(pos_list["y"])
         if input_graph.renumbered is True:
+            if len(input_graph.renumber_map.implementation.col_names) > 1:
```

Comment:
This reference: `input_graph.renumber_map.implementation.col_names` exposes internal implementation wherever we use it.  I'd suggest adding a data member or method (either on the NumberMap or Graph class -- I'm inclined to think the latter) to indicate that it's a multi-column situation.  Otherwise if we eventually change how we implement multi-column (i.e. when we rework it to use an indirect hash table as we have discussed) then all references to the implementation of renumber_map will need to be changed throughout the python code.